### ISSUE TITLE
can you give a sample of the training jsonl file you used?

### ISSUE THREAD

**Issue by tairan-w** [chaoyi-wu/PMC-LLaMA#5]
The original jsonl file from s2orc does not fit in your code. Thanks!

**Comment by anonymoususerr1** [comment]
Can you further explain how you managed to obtain a jsonl file via s2orc?

**Comment by jcrangel** [comment]
Yes I also would like to know more about the pretrain process. For example the code `finetune_pp_peft_trainer_paper_deepspeed.py` in `pretrain.sh` is not included.
